src/components/ReviewCardButtons.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import { patchReviewVotes } from "../api";
+
+const ReviewCardButtons = ({ review_id, setReviewVotes }) => {
+    const [incVotes, setIncVotes] = useState(0);
+
+    const upVoteHandler = (effect) => {
+        setIncVotes(1);
+        patchReviewVotes(review_id, incVotes)
+            .then((data) => {
+                setReviewVotes(data);
+            })
+            .catch((err) => {
+                console.log(err);
```

Comment:
Inside the .catch, we should be doing undoing our optimistic rendering to show that the vote didn't go through. If you made it look like the vote had increased by one, this is where you would undo that to show that it had been unsuccessful